templates/profile/prometheus/exporter/node/rubyHealthCheck-testall.sh.erb

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+RBENVVER="$(rbenv version)"
+
+if [ $? -ne 0 ]; then
+  echo "rbenv version can't run... source the profile setup before trying to test"
+  exit 1
+fi
+#./versions.sh | xargs -I{} ./testruby.sh {} | tee -a report.txt
+
+echo '# HELP ruby_heath_check Check Health of Ruby Version'
+echo '# TYPE ruby_health_check gauge'
+
+for VER in $(./versions.sh)
```

Comment:
We can inline the versions.sh here and save some puppet code. It was standalone for convenience while putting the pieces together, but it's not really useful as such for the long term.

Reply:
I think I corrected this.  In testing its working.